### DISCUSSION TITLE
[2024-09-30 09:32:07] ERROR root job - start [main]:  Failed to start backup: [checkpoint inconsistent: missing or broken bitmap 'virtnbdbackup.18' for disk 'vda']

### DISCUSSION THREAD

**Discussion by linuxgokan** [abbbi/virtnbdbackup#226] [General]
Hello

I am getting the following error on the virtual machines that I regularly back up. What is the reason?

[2024-09-30 09:32:07] ERROR root job - start [main]:  Failed to start backup: [checkpoint inconsistent: missing or broken bitmap 'virtnbdbackup.18' for disk 'vda']

**Comment by abbbi** [comment]
i dont know. What happened between backup 17. and 18.
The bitmap created prior now required for incremental backup is missing from the qcow files.

**What happened to those qcow files between backups? They must have been changed by third party.**

Please read the following articles of the README:

https://github.com/abbbi/virtnbdbackup?tab=readme-ov-file#transient-virtual-machines-checkpoint-persistency-on-clusters
https://github.com/abbbi/virtnbdbackup?tab=readme-ov-file#backup-fails-with-unable-to-execute-qemu-command-transaction-bitmap-already-exists
https://github.com/abbbi/virtnbdbackup?tab=readme-ov-file#backup-fails-with-error-during-checkpoint-removal-internal-error-bitmap-xx-not-found-in-backing-chain-of-xx

As you receive this error so often you do something to your environment that conflicts with a propper backup operation.

this is not an issue in the program itself but rather how you manage your environment.

**Comment by tuomotalvitie** [comment]
I once eneded up with the situation where all the vms were missing one bitmap as the destination run out of allocated storage space (The version used was 2.9). After removing the metadata of the latest attempt (and one actual checkpoint to be sure) virtnbdbackup once again worked properly.
